File: app/src/main/java/com/uce/kotlinclassic/MainActivity.java
package com.uce.kotlinclassic;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

//    private fun load() {
//        val path = Paths.get("words.txt")
//        val t = measureTimeMillis {
//            Files.lines(path).parallel().forEach { line ->
//                    val words = line.substringAfter('\t').replace('\t', ' ').toLowerCase()
//                val freq = line.substringBefore('\t').toInt()
//                ngrams.add(NGram(words, freq))
//            }
//        }
//        println("$currentThread: Loaded file in $t msec")
//    }
}

File: app/src/main/java/com/uce/kotlin/MainActivity.kt
package com.test.kotlin

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import kotlinx.android.synthetic.main.activity_main.*
import com.test.kotlin.Delegate.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        // Example of a call to a native method
        sample_text.text = "测试哦"
        sample_text.setOnClickListener {
            //            Test.main()
//            Test2.main()
            main7()
        }
    }
}
